verify_project.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 set -e
 
-py.test --cov=demo_python_at
+py.test --cov=demo_python_at demo_python_at
```

Comment:
@ivan-podorozhnyi we have to run unit tests (not regular tests). And commit's subject does not explain the problem it's solving.